images/win/scripts/Installers/Install-Git.ps1

```diff
@@ -3,33 +3,18 @@
 ##  Desc:  Install Git for Windows
 ################################################################################
 
-function getSimpleValue([string] $url, [string] $filename ) {
-    $fullpath = "${env:Temp}\$filename"
-    Invoke-WebRequest -Uri $url -OutFile $fullpath
-    $value = Get-Content $fullpath -Raw
-
-    return $value
-}
-
-# Install the latest version of Git for Windows
-$gitTag = getSimpleValue -url "https://gitforwindows.org/latest-tag.txt" -filename "gitlatesttag.txt"
-$gitVersion = getSimpleValue -url "https://gitforwindows.org/latest-version.txt" -filename "gitlatestversion.txt";
-
-$installerFile = "Git-$gitVersion-64-bit.exe";
-$downloadUrl = "https://github.com/git-for-windows/git/releases/download/$gitTag/$installerFile";
-Install-Binary  -Url $downloadUrl `
-                -Name $installerFile `
-                -ArgumentList (
-                    "/VERYSILENT", `
-                    "/NORESTART", `
-                    "/NOCANCEL", `
-                    "/SP-", `
-                    "/CLOSEAPPLICATIONS", `
-                    "/RESTARTAPPLICATIONS", `
-                    "/o:PathOption=CmdTools", `
-                    "/o:BashTerminalOption=ConHost", `
-                    "/o:EnableSymlinks=Enabled", `
-                    "/COMPONENTS=gitlfs")
+Choco-Install -PackageName git -ArgumentList (
+    "--installargs", `
+    "/VERYSILENT", `
+    "/NORESTART", `
+    "/NOCANCEL", `
+    "/SP-", `
+    "/CLOSEAPPLICATIONS", `
+    "/RESTARTAPPLICATIONS", `
+    "/o:PathOption=CmdTools", `
+    "/o:BashTerminalOption=ConHost", `
+    "/o:EnableSymlinks=Enabled", `
+    "/COMPONENTS=gitlfs")
```

Comment:
Should we get rid of `--ignore-dependencies` flag for hub in this case?

Reply:
@maxim-lobanov yes, the ignore-dependencies can now be safely removed, as the required dependencies will now be in place.

Reply:
Thank you, `--ignore-dependencies` is removed now